### PR TITLE
J154: * icons for title page

### PR DESCRIPTION
**Pull request by EugeniusGR**
image

### CODE REVIEW

File: frontend/public/manifest.json [hunk 1]
```diff
@@ -9,12 +9,12 @@
   "description": "Jabber Application",
   "icons": [
     {
-      "src": "/img/icons/icon-512.png",
+      "src": "/img/icons/icon-512-pink.png",
```
Comment (EugeniusGR) [inline frontend/public/manifest.json]: Wrong image name

Comment (EugeniusGR) [reply frontend/public/manifest.json]: Will fix it soon

Comment (what1s1ove) [reply frontend/public/manifest.json]: remove old, and use only these

Comment (what1s1ove) [reply frontend/public/manifest.json]: you also need to change apple icon, social icon, favicon, svg icon

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Comment by EugeniusGR** [comment]
image
image
image

**Review by what1s1ove — CHANGES_REQUESTED** [review]
social icon should be fully pink

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**